### ISSUE TITLE
clearHistory on if navigation

### ISSUE THREAD

**Issue by uzarsalan** [halfnelson/svelte-native#96]
Hi! I'm using such layout

```javascript
<page actionBarHidden="true">
  <frame id="rootFrame">
      {#if loggedIn}
       <MainPage />
      {:else}
        <LoginPage />
      {/if}
  </frame>
</page>
```

when logged in changes from false to true frame navigates to MainPage.
How can i clear history stack for prevent user to go LoginPage, pressing back?

**Comment by halfnelson** [comment]
The child element of Frame was designed as a shortcut for setting the "defaultPage" property of the Frame component, and I hadn't really figured on it changing at runtime. I had assumed all subsequent calls would be via `Navigate` etc. However this is an excellent use case and probably just involves me adding the `clearHistory` attribute on the navigation that takes place under the hood. 
https://github.com/halfnelson/svelte-native/blob/master/src/dom/native/FrameElement.ts#L31

I'll leave this ticket open until I get around to implementing this. In the short term , you should instead try to use the `clearHistory` option ( `navigate( { page: LoginPage, clearHistory: true })` )

**Comment by halfnelson** [comment]
shipped in svelte-native 0.7.0
